### DISCUSSION TITLE
Add Twitter Card support to Yourls shortlinks

### DISCUSSION THREAD

**Discussion by HeySteets** [YOURLS/YOURLS#2995] [Q&A] [answered]
Hi there! 
I'm setting up a new installation of YOURLS, and I wanted to know if there was a way to enable Twitter Card support. All I need to do is add some <meta> tags to each short link, which should pull some link metadata like title and a preview image, possibly user-specifiable. Is there any way this can be accomplished with a plugin? I've browsed around and inspected the YOURLS code a bit, but I couldn't find any obvious place that custom HTML could be injected to a redirected link.
Thank you!

**Comment by dgw** [comment, marked as answer]
> but I couldn't find any obvious place that custom HTML could be injected to a redirected link.

That's because there isn't one. YOURLS doesn't send HTML with a redirection; without any plugins to modify the behavior, visited short links are just a HTTP 301 Moved Permanently status code + a `Location` header with the long URL for the client to follow. It's up to the final destination (whatever site YOURLS is redirecting to) to provide metadata for Twitter, Facebook, etc.

**Reply by HeySteets** [reply]
I pretty much assumed it'd just be generating 3XX response codes. Just wanted to double check there isn't another way.

Thank you for the fast response!
